docs/UserGuide.adoc

```diff
@@ -69,10 +70,10 @@ Examples:
 
 * `add \n Two Chefs Eating Place \a 116 Commonwealth Cres, #01-129 \c Chinese`
```

Comment:
u can change the example to include sth with tags too!! hahaha here's an example so u don't need think: add \n Koma \a Marina Bay Sands, 2 Bayfront Ave, #B1-67 \c Japanese \t expensive`